packages/dashboard/README.md

```diff
@@ -0,0 +1,24 @@
+# Dashboard
+
+The developing new dashboard.
+
+Expected to working in three mode:
+
+## Storybook
+
+Most of the components should be expected to work in the Storybook.
+This means you should split logic and UI components.
+
+## Isolated website
+
+This project should be able to run under http or https instead of chrome-extension.
+
+Currently, you must also install the development mode (`./dist/`) Mask Network extension where the extension ID must be `jkoeaghipilijlahjplgbfiocjhldnap`. Otherwise you will encountered error "Error: Attempting to use a disconnected port object".
```

Comment:
```suggestion
Currently, you must also install the development mode (`./dist/`) Mask Network extension where the extension ID must be `jkoeaghipilijlahjplgbfiocjhldnap`.
Otherwise you will encountered error "Error: Attempting to use a disconnected port object".
```